l5kit/setup.py

```diff
@@ -31,9 +31,9 @@
         "ipywidgets"
     ],
     extras_require={
-        "dev": ["pytest==5.4.3", "mypy==0.782", "setuptools", "twine", "wheel", "pytest-cov==2.10.0", "flake8==3.8.3",
-                "black==19.10b0", "isort==5.0.4", "Sphinx==3.1.1", "sphinx-rtd-theme==0.5.0", "recommonmark==0.6.0",
-                "pre-commit==2.5.1"]
+        "dev": ["pytest>=5.4.3", "mypy>=0.782", "setuptools", "twine", "wheel", "pytest-cov>=2.10.0", "flake8>=3.8.3",
```

Comment:
why do you need `mypy>=0.782` and "pytest-cov>=2.10.0" and "flake8>=3.8.3" and "isort>=5.0.4" ?
These are dev packages and you should be able to always use the latest version right? 
Is there any specific thing happening with the pinned versions?

Reply:
There are issues happening with previous versions (so for example mypy 0.77). I guess this is not really necessary though, as the behaviour should be the same as `*`. I'll remove it